src/components/Footer/Footer.jsx

```diff
@@ -8,43 +9,51 @@ const Footer = () => {
           <h4>Quick Links</h4>
           <ul className="footer-nav-links">
             <li>
-              <a href="#">Home</a>
+              <Link to="/">Home</Link>
             </li>
             <li>
-              <a href="#">Products</a>
+              <Link to="/products">Products</Link>
             </li>
             <li>
-              <a href="#">My Cart</a>
+              <Link to="/">My Cart</Link>
             </li>
             <li>
```

Comment:
import these components into footer.jsx to make these links working

Reply:
I haven't started the cart page yet. I will add the links later